11.0.Dockerfile

```diff
@@ -36,6 +41,11 @@ ONBUILD ENV ADMIN_PASSWORD="$ADMIN_PASSWORD" \
             PGDATABASE="$PGDATABASE" \
             PROXY_MODE="$PROXY_MODE" \
             SMTP_SERVER="$SMTP_SERVER" \
+            SMTP_PORT=$SMTP_PORT \
```

Comment:
Is there any reason why this has no surrounding quotes? I'd put them, just for consistency.